examples/jenkins/pipeline/jenkinstemplate.json

```diff
@@ -37,6 +37,10 @@
       "apiVersion": "v1",
       "metadata": {
         "name": "${JENKINS_SERVICE_NAME}",
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
```

Comment:
Ugh, I'm really concerned about not using Kinds.  Most clients are kind
focused when it comes to doing references.

On Mon, Jun 13, 2016 at 11:01 AM, Ben Parees <notifications@github.com>
wrote:

> In examples/jenkins/pipeline/jenkinstemplate.json
> <https://github.com/openshift/origin/pull/9189#discussion_r66807502>:
>
> > @@ -37,6 +37,10 @@
> >        "apiVersion": "v1",
> >        "metadata": {
> >          "name": "${JENKINS_SERVICE_NAME}",
> > +        "annotations": {
> > +          "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
>
> we care about the resource type so we know where to go look for
> jenkins-jnlp object.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9189/files/1594d43a831b62264181448927589cd7e7124d01#r66807502>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p4NqFzla3a-TE_nh7xm6Zmv5mT--ks5qLXDfgaJpZM4IvZXy>
> .
>